app/src/main/res/xml/filepaths.xml

```diff
@@ -0,0 +1,5 @@
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <files-path
+        name="mFiles"
```

Comment:
Also, how does this work? I'm just curious because I've never seen this before!

Reply:
This is the filepath I had to define for passing information from the app to an email app via the Content Provider. We don't really actively use it anywhere - it's more a background thing.